rajawali/src/main/java/org/rajawali3d/Geometry3D.java

```diff
@@ -263,22 +263,28 @@ public void addFromGeometry3D(Vector3 offset, Geometry3D geometry, boolean creat
         float[] newColors = null;
         float[] newTextureCoords = null;
         int[] newIntIndices = null;
-        float[] mVerticesArray = null;
-        float[] mNormalsArray = null;
-        float[] mColorsArray = null;
-        float[] mTextureCoordsArray = null;
-        int[] mIndicesArray = null;
+        float[] mVerticesArray = new float[0];
```

Comment:
We have to initialise these values to prevent calls to `concatAllFloat()` failing with NullPointerExceptions.